utils/nctl/sh/contracts-transfers/do_dispatch_native.sh

```diff
@@ -70,19 +73,24 @@ function main()
                 --secret-key "$CP1_SECRET_KEY" \
                 --amount "$AMOUNT" \
                 --target-account "$CP2_ACCOUNT_KEY" \
-                --transfer-id $((DISPATCHED + 1)) \
-                | jq '.result.deploy_hash' \
-                | sed -e 's/^"//' -e 's/"$//'
+                --transfer-id $((DISPATCHED + 1))
```

Comment:
This should be `DISPATCH_ATTEMPTS` since `DISPATCHED` was removed.

Reply:
Good catch!  Fixed in [d4da077](https://github.com/casper-network/casper-node/pull/3907/commits/d4da077e2e77bc1c24f30ea0d3578534a45eee7d).